package.json

```diff
@@ -75,11 +78,6 @@
           "varsIgnorePattern": "React"
         }
       ]
-    },
-    "overrides": {
-      "extends": [
-        "xo-typescript"
-      ]
```

Comment:
This is what happens if I do this.
cc @sindresorhus
```
  test.js
  ✖  Parsing error: "parserOptions.project" has been set for @typescript-eslint/parser.
The file does not match your project config: test.js.
The file must be included in at least one of the projects provided.  
```

Reply:
Might have been fixed in https://github.com/xojs/xo/releases/tag/v0.28.1

Reply:
Sadly not

Reply:
Your other comment fixed it though! https://github.com/vadimdemedes/dom-chef/pull/56/commits/3f34d9d824d3caf25aba2dec4202270c2f34a42a